IBC_WORKING_GROUP.md

```diff
@@ -0,0 +1,33 @@
+# IBC Working Group
+
+Join and contribute to the IBC working group 
+
+The evolution of IBC requires cross-industry collaboration to support developing standards that are secure, open, and deployable. The IBC working group is a growing team of contributors supporting the standardization of blockchain development from protocol level to applications. 
+
+Join the IBC [Working Group](https://zc.vg/sf/giAEr) email updates
+
+Join the IBC Telegram channel
```

Comment:
I am not a huge fan of using Telegram to coordinate or discuss, it is too ephemeral.

I would prefer a Discourse server, I think forums are much more conducive to long-form, considered discussion.

Is this something that would be possible? I would be quite willing to help set it up, administer/moderate, etc.

Perhaps a dedicated subsection of [forum.cosmos.network](https://forum.cosmos.network) could also be used, although I think there are advantages in branding it separately.